docs/plugins.md

```diff
@@ -92,3 +92,15 @@ You need to register a `plugin` only once per JSS instance. There is a number of
       }
     })
     ```
+
+1. Hook `onUpdate(data, rule, sheet)`.
```

Comment:
I was keeping it undocumented, because it was used only internally, but yeah, once we start using it outside, we should document it.